modules/Core/src/main/java/org/terasology/logic/inventory/InventoryUIClientSystem.java

```diff
@@ -69,4 +91,121 @@ public void onToggleInventory(InventoryButton event, EntityRef entity, ClientCom
             // do not consume the event, so that the inventory will still open
         }
     }
+
+
+    /*
+      The numbersBetween() and getTransferEntity() methods were
```

Comment:
Even if this is a private method still use the `/**` javadoc approach instead of a multi-line plain comment. Javadoc is nice! And usually in an IDE if you `/**<enter>` on a method, field, or class the IDE will auto-generate a Javadoc block for you including return values and parameters. 

Don't be shy about making lines longer, our max line length is like 175 but for comments you could try to stay below 120 or so.